Makefile

```diff
@@ -34,8 +34,11 @@ watch:
 unit-test: build/test/background.js build/test/ui.js build/test/shared-utils.js
 	$(KARMA) start karma.conf.js
 
-shared/content-scope-scripts: node_modules/@duckduckgo/content-scope-scripts
-	cp -r node_modules/@duckduckgo/content-scope-scripts shared/
+shared/content-scope-scripts: node_modules/@duckduckgo/content-scope-scripts $(shell find node_modules/@duckduckgo/content-scope-scripts/src -type f)
```

Comment:
Can we not remove this copying entirely? (I thought we already had.)

Reply:
I did try to update imports in the unit-test to get the files direct from node_modules, but this breaks due to the mixing of `require` and `import` statements. Ideally, we can fix the build for the tests and switch to `import` everywhere, then this copying can be removed.

Note that this is the only place where we need to copy, I removed the ones during the grunt deprecation.